code/parachain/runtime/dali/src/lib.rs

```diff
@@ -1196,6 +1211,157 @@ impl cosmwasm::Config for Runtime {
 	type WeightInfo = cosmwasm::weights::SubstrateWeight<Runtime>;
 }
 
+parameter_types! {
+	pub const RelayChainId: RelayChain = RelayChain::Rococo;
+	pub const SpamProtectionDeposit: Balance = 1_000_000_000_000;
+	pub const MinimumConnectionDelay: u64 = 0; // well we don't need the security tbh.
```

Comment:
why we do not need it?

Reply:
Oh we'll definitely need it in production

Reply:
@vmarkushin Can this be commented better? (Capitalized, punctuated, explaining why 0 is acceptable for Dali but not acceptable for Production). As well as, is there a reason to use unsafe configurations in Dali, instead of just having correct configurations everywhere?

Reply:
@seunlanlege could you answer @KaiserKarel's comment?

Reply:
@KaiserKarel @seunlanlege and @Wizdave97 are unavailable at the moment. I can set it to 5 minutes for now as it's done in the [docs](https://github.com/ComposableFi/centauri/blob/b97264308da51e67e0002e6706fab741a6a996a6/contracts/pallet-ibc/README.md?plain=1#L31) and leave a TODO to refine the parameter later, how about that?

Reply:
imho any NON zero is good to check setting it at all works.